modules/odc-adding-health-checks.adoc

```diff
@@ -3,17 +3,17 @@
 // applications/application-health
 
 :_mod-docs-content-type: PROCEDURE
-[id="odc-adding-health-checks"]
+[id="odc-adding-health-checks_{context}"]
 = Adding health checks using the Developer perspective
 
-You can use the *Topology* view to add health checks to your deployed application.
+You can use the *Topology* view to add health checks to your deployed application. 
 
 .Prerequisites:
 * You have switched to the *Developer* perspective in the web console.
 * You have created and deployed an application on {product-title} using the *Developer* perspective.
 
 .Procedure
-. In the *Topology* view, click on the application node to see the side panel. If the container does not have health checks added to ensure the smooth running of your application, a *Health Checks* notification is displayed with a link to add health checks.
+. In the *Topology* view, click the application node to see the side panel. If the container does not have health checks added, a *Health Checks* notification is displayed with a link to add health checks.
```

Comment:
is "application node" a button? It says "click the application node" which makes me think it is a button and should be bold.